management/server/account.go

```diff
@@ -2206,3 +2226,39 @@ func separateGroups(autoGroups []string, allGroups map[string]*nbgroup.Group) ([
 	}
 	return newAutoGroups, jwtAutoGroups
 }
+
+func (am *DefaultAccountManager) GetAccountWithBackpressure(ctx context.Context, accountID string) (*Account, error) {
+	req := &AccountRequest{
+		AccountID:  accountID,
+		ResultChan: make(chan *AccountResult, 1),
+	}
+
+	log.WithContext(ctx).Debugf("requesting account with backpressure: %s", accountID)
+	startTime := time.Now()
+	am.getAccountRequestCh <- req
```

Comment:
I am not sure about the teardown procedure of the Management service but if you handle the cancel context in the `processGetAccountRequests` then should handle the here too.